scripts/install-exec-dependencies.sh

```diff
@@ -17,18 +17,20 @@ fi
 
 # Download ssm agent static binaries in BINARY_PATH
 mkdir -p /tmp/ssm-binaries && cd /tmp/ssm-binaries
+
+gpg --import /tmp/amazon-ssm-agent.gpg
+
 case $ARCHITECTURE in
 'x86_64')
     curl -fLSs "https://amazon-ssm-${REGION}.s3.${REGION}.amazonaws.com${host_suffix}/${EXEC_SSM_VERSION}/linux_amd64/amazon-ssm-agent-binaries.tar.gz" -o amazon-ssm-agent.tar.gz
-    echo "94be5ddec82d67d2f799d2fd1c8ab3f597e5d166b9750891a135d3093e15aa24 ./amazon-ssm-agent.tar.gz" >./amazon-ssm-agent.tar.gz.sha256
```

Comment:
we still need these shasums -- ecs-agent uses these to be sure the tar hasn't been tampered with.

Reply:
We are probably not using it? I see the following files being copied to binary path - https://github.com/aws/amazon-ecs-ami/blob/main/scripts/install-exec-dependencies.sh#L35-L37